frontend/src/Profile/Profile.js

```diff
@@ -25,7 +49,6 @@ const UserProfile = () => {
       </div>
     );
   }
-
   const profileImageUrl =
     userProfile.images && userProfile.images.length > 0 ? userProfile.images[0].url : defaultAvatar;
 
```

Comment:
line 75: link is taking to an old end point should be changed to /detailed-stats